hypothesis-python/tests/cover/test_fuzz_one_input.py

```diff
@@ -155,3 +158,15 @@ def test(s):
     (saved_examples,) = db.data.values()
     assert seen == buffers
     assert len(saved_examples) == db_size
+
+
+def test_fuzzing_invalid_test_raises_error():
+    # Invalid: @given with too many positional arguments
+    @given(st.integers(), st.integers())
+    def invalid_test(s):
+        pass
+
+    with raises(InvalidArgument) as e:
```

Comment:
```suggestion
    with pytest.raises(InvalidArgument, match="Too many positional arguments"):
```

See [`pytest.raises` docs](https://docs.pytest.org/en/latest/reference/reference.html#pytest-raises); I think we can use this in each case for somewhat clearer tests.